app/webpacker/styles/components/breadcrumb.scss

```diff
@@ -0,0 +1,49 @@
+.breadcrumb {
+  ol {
+    display: flex;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    padding: 0.5rem $indent-amount;
+    margin: 3em 0 0 0;
+    list-style: none;
+    * {
+      box-sizing: border-box;
+    }
+    li {
+      a {
+        position: relative;
+        display: block;
+        margin-right: 0.8em;
+        color: black;
+        font-size: 0.842em;
+          @include mq($until: tablet) {
+            font-size: 0.737em;
+          }
+        &:after {
+          content: "";
+          display: inline-block;
+          width: 7px;
```

Comment:
I think the arrow size should reflect the font-size - so this should probably use `em` units to reflect the sizing of the breadcrumb font - likewise the border size

An alternative might be to use `&raquo;` or `&laquo;` in `content` but that might not achieve the desired visual style